package.json

```diff
@@ -9,11 +9,12 @@
     "access": "public"
   },
   "scripts": {
-    "bundle": "bash ./scripts/build_latest_bundle.sh",
     "build": "npx tsc --project tsconfig.build.json",
-    "rollup": "npx rollup -c rollup.config.js",
+    "bundle": "bash ./scripts/build_latest_bundle.sh",
+    "lint:fix": "npx eslint . --ext .ts --fix",
     "lint": "npx eslint . --ext .ts",
-    "lint:fix": "npx eslint . --ext .ts --fix"
+    "rollup": "npx rollup -c rollup.config.js",
+    "upgrade": "bash ./scripts/build_latest_bundle.sh"
```

Comment:
I sorted these lines and added "upgrade" for quickly upgrading to use the `@netacea/akamai` version